packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -402,6 +411,99 @@ describeCompat("Summaries", "NoCompat", (getTestObjectProvider) => {
 		await provider.ensureSynchronized();
 		await summaryCollection.waitSummaryAck(container.deltaManager.lastSequenceNumber);
 	});
+
+	async function getNackProm(container: IContainer) {
+		return new Promise<void>((resolve, reject) => {
+			const callback = (_reason, error) => {
+				try {
+					assert.strictEqual(error?.statusCode, 429);
+					assert.strictEqual(
+						error?.message,
+						"Nack (ThrottlingError): Submit a summary before inserting additional operations",
+					);
+					resolve();
+				} catch (err) {
+					reject(err);
+				} finally {
+					container.deltaManager.off("disconnect", callback);
+				}
+			};
+			container.deltaManager.on("disconnect", callback);
+		});
+	}
+
+	it("Can summarize after hitting nack on unsummarized ops", async function () {
+		if (provider.driver.type !== "local") {
+			this.skip();
+		}
+		const stringId = "sharedStringKey";
+		const registry: ChannelFactoryRegistry = [[stringId, SharedString.getFactory()]];
```

Comment:
Is this typically necessary?  In e2e tests I've worked on I haven't specified the registry